FileUploader: Add Getting Started topic (#7693)
DevExpress/devextreme-documentation: concepts/05 UI Components/FileUploader/10 Getting Started with FileUploader/20 Customize FileUploader.md

> You can customize FileUploader visual elements as your needs dictate. To customize the file drag and drop zone, specify the [dropZone](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#dropZone) property. To configure component texts, specify properties such as [labelText](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#labelText), [readyToUploadMessage](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#readyToUploadMessage), and [uploadButtonText](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#uploadButtonText).

Reviewer: ```suggestion
You can customize FileUploader visual elements as your needs dictate. To customize the file drag and drop zone, specify the [dropZone](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#dropZone) property. To configure display text, specify properties such as [labelText](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#labelText), [readyToUploadMessage](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#readyToUploadMessage), and [uploadButtonText](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#uploadButtonText).
```

This example implements a custom drop zone and configures the **labelText** property:

---

##### jQuery

    <!-- tab: index.js -->
    $(function() {
        $("#file-uploader").dxFileUploader({
            labelText: 'Drop Images Here to Upload...',
            dropZone: '.file-uploader-block',
        });
    });

##### Angular

    <!-- tab: app.component.html -->
    <dx-file-uploader
        labelText="Drop Images Below to Upload..."
        dropZone=".file-uploader-block"
    ></dx-file-uploader>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxFileUploader
            label-text="Drop Images Below to Upload..."
            drop-zone=".file-uploader-block"
        />
    </template>

##### React

    <!-- tab: App.tsx -->
    export default function App(): JSX.Element {
        return (
            <FileUploader
                labelText="Drop Images Below to Upload..."
                dropZone=".file-uploader-block"
            />
        );
    }

---

> To customize FileUploader visual elements further, you can configure custom CSS styles. This example implements styles to position the **labelText** string below the "Select a file" button. Additionally, this example centers the component horizontally inside the FileUploader drop zone. Scrollbars appear when the file list overflows the drop zone container (`overflow-y: auto;`).

Reviewer 2: ```suggestion
To customize FileUploader visual elements further, use custom CSS. This example implements styles to position the **labelText** string below the "Select a file" button. Additionally, this example centers the component horizontally inside the FileUploader drop zone. Scrollbars appear when the file list overflows the drop zone container (`overflow-y: auto;`).
```

> To customize FileUploader visual elements further, you can configure custom CSS styles. This example implements styles to position the **labelText** string below the "Select a file" button. Additionally, this example centers the component horizontally inside the FileUploader drop zone. Scrolling is enabled when uploaded files overflow the drop zone container with the `overflow-y: auto;` style.

Reviewer: ```suggestion
To customize FileUploader visual elements further, you can configure custom CSS styles. This example implements styles to position the **labelText** string below the "Select a file" button. Additionally, this example centers the component horizontally inside the FileUploader drop zone. Scrollbars appear when the file list overflows the drop zone container (`overflow-y: auto;`).
```
